template/tests/index.test.js

```diff
@@ -0,0 +1,17 @@
+import { h } from "preact";
+import { shallow } from 'enzyme';
+
+import Hello from "../src/component";
+
+describe("Hello logic", () => {
+  test("should be able to run tests", () => {
+    expect(1 + 2).toEqual(3);
+  });
+});
```

Comment:
don't think we need this test :p

Reply:
I'm hoping I copy/pasted that *wonderful and super useful* test from somewhere.... will remove it